### ISSUE TITLE
Build fails (incompatible type as argument)

### ISSUE THREAD

**Issue by thorwald** [appc/docker2aci#79]
Getting this when installing

```
go get github.com/appc/docker2aci
```

```
# github.com/appc/docker2aci/lib/common
go/src/github.com/appc/docker2aci/lib/common/common.go:96: cannot use reader (type *aci.TarReadCloser) as type *tar.Reader in argument to aci.ValidateArchive
go/src/github.com/appc/docker2aci/lib/common/common.go:358: cannot use *reader (type aci.TarReadCloser) as type tar.Reader in argument to tarball.Walk
```

**Comment by jeromegn** [comment]
Facing the same issue just now.

**Comment by jonboulle** [comment]
Hmm, this is a result of recent appc changes. @sgotti would you mind taking a look?

**Comment by sgotti** [comment]
@jonboulle. Sure. Didn't had time today, will do tomorrow.

**Comment by sgotti** [comment]
@jonboulle I wrote of these needed changes in rkt and docker2aci in appc/spec#461 but forgot that docker2aci doesn't use vendored libraries... Created #80.

**Comment by thorwald** [comment]
Tried you patch and it works, thanks!

**Comment by jonboulle** [comment]
Fixed by #80
